pkg/cassandra/session.go

```
@@ -181,6 +181,25 @@ func (s *Session) Close() {
 	s.session.Close()
 }
 
+// reachableHosts opens a session against every contact point and returns the
+// ones that answered, so the config page can tell the user which of the hosts
+// it was given are actually usable.
+func reachableHosts(hosts []string, dial func(string) (*Session, error)) []string {
```

golangci-lint (2.12.2)

[error] func reachableHosts is unused

(unused)